fetcher/tar.py

```diff
@@ -99,18 +101,26 @@ def unpack(self, downloadFirst=False):
         update = tarfile.open(name=self.workspacefile, mode='r:*')
         update.extractall(self.workspaceunpack)
```

Comment:
There is a problem too in extracting the boot files directly from the tar to the boot partition: handling atomicity which is currently handled as a copy method in resinhup.